docs/src/main/sphinx/glossary.rst

```diff
@@ -0,0 +1,199 @@
+========
+Glossary
+========
+
+The glossary contains a list of key Trino terms and definitions.
+
+Terms A-E
+---------
```

Comment:
I question the value of these headers. This file just really isn't big enough to warrant them, in my opinion. Alphabetical sorting should be enough.

Reply:
Yeah, we certainly want to fill these in at some point but for now yeah, it's kind of pointless. I'll keep them there and let's get @electrum's POV on it.

Reply:
In Trino we don't keep stuff that's not used, so I'd remove them.

Reply:
Cool, well it seems like we're all in agreement here. I'll remove them and keep this conversation open in case @electrum disagrees.